### PR TITLE
submission

### PR DESCRIPTION
**Pull request by jlaurentpdx**

### CODE REVIEW

File: src/components/GuestForm/GuestForm.jsx [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
-    const newEntry = { id, name, comment };
-
-    setEntries([...entries, newEntry]);
-    setUser(name);
-    setComment('');
+    try {
+      const newEntry = await createEntry(entry);
+      setUser(name);
+      setComment('');
+      await setEntries((prevState) => [newEntry[0], ...prevState]);
```
Comment (gabrielsimek) [inline src/components/GuestForm/GuestForm.jsx]: We shouldn't need to await setStates

File: src/context/ThemeContext.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      localStorage.setItem('theme', 'dark');
+      setTheme('dark');
+    } else {
+      localStorage.setItem('theme', 'light');
+      setTheme('light');
+    }
+  };
+
+  useEffect(() => {
+    const localTheme = localStorage.getItem('theme');
```
Comment (gabrielsimek) [inline src/context/ThemeContext.jsx]: We could possibly move this logic inside of the useState on line 6

Comment (jlaurentpdx) [reply src/context/ThemeContext.jsx]: Thank you! The block felt a bit clunky to me when writing, too, so this suggestion is especially helpful. Dropped the `useEffect` and refactored the redundancies in the `toggleTheme` in [`faa5f65` (#3)](https://github.com/jlaurentpdx/guestbook/pull/3/commits/faa5f65ec7532e62ceedabe14aad3c4e167c483a).

File: src/components/GuestForm/GuestForm.jsx (hunk 1 above)
Comment (jlaurentpdx) [reply src/components/GuestForm/GuestForm.jsx]: Good catch! Corrected in [`faa5f65` (#3)](https://github.com/jlaurentpdx/guestbook/pull/3/commits/faa5f65ec7532e62ceedabe14aad3c4e167c483a)

**Pull request merged**